recipes/gwtools/meta.yaml

```
@@ -0,0 +1,53 @@
+{% set name = "gwtools" %}
+{% set version = "1.0.2" %}
+{% set sha256 = "ac413474d5ac3ddfbca21632d224a68a83b0a662ebdd215cb7e0391b92b79d40" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: "{{ sha256 }}"
```

To make life easier for the update bot:
```suggestion
  sha256: ac413474d5ac3ddfbca21632d224a68a83b0a662ebdd215cb7e0391b92b79d40
```